studio/schemas/documents/compensations.ts

```diff
@@ -19,7 +20,14 @@ const compensations = defineType({
         "Enter the primary title that will be displayed at the top of the compensation page. This is what users will see when they visit the page.",
     },
     titleSlug,
-    // add pension here. pension doesn't rely on locations
+    defineField({
+      name: "showSalaryCalculator",
+      title: "Show Salary Calculator",
+      description: "Should the salary calculator be visible on the page?",
```

Comment:
```suggestion
      description: "Toggle this setting to show or hide the salary calculator on the page.",
```

Should we maybe add some explanation as to what that calculator is?? Like that it's a design feature that calculates the estimated salary? 🤷‍♀️